torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -306,3 +309,82 @@ def _create_onnx_friendly_decomposition_table() -> (
 DEFAULT_ONNX_EXPORTER_DECOMPOSITION_TABLE: Dict[
     torch._ops.OpOverload, Callable
 ] = _create_onnx_friendly_decomposition_table()
+
+
+def get_symbolic_function(
```

Comment:
note: this part is extracting dispatch code from `fx_to_onnxscript.py` to `function_dispatcher.py`. The goal is to make it re-usable for the analysis pass. A proper dispatcher refactor is still required, and is not the scope of this PR.

Reply:
#100660 targets the refactor of `function_dispatcher`